ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -417,7 +421,8 @@ export default defineComponent({
 
       // Store a snapshot of the existing OwnerGroups for baseline of current state
       await setMhrTransferCurrentHomeOwnerGroups(cloneDeep(data.ownerGroups))
-
+      // Store existing submitting party to be used if user is BC Registry Staff
+      setMhrTransferSubmittingParty(data.submittingParty)
```

Comment:
O i see, i guess this answers my question from above, this is setting the submitting party to store from the existing Reg?

Reply:
Oh bit confused, you are setting the submitting party down below in the `parseSubmittingParty` 
Why here too?

Reply:
Yes..